<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_table</class>
 <widget class="QWidget" name="credit_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="info_refills">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Refills:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="info_rate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Tax:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="info_gains">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Capital gains:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="info_withdrawals">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Withdrawals:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="info_fullsum">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Amount+interest:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="info_tax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Gain with tax:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget::item:selected {
	background-color: rgb(175, 97, 33);
	color: rgb(255, 255, 255);
}

QTableWidget {
    border-radius: 10px;
    padding: 0 8px;
	background-color: rgb(255, 255, 255);
	color:black;
	color: rgb(0, 0, 0);
	gridline-color: rgb(0, 0, 0);
	border: 1px solid black;
}</string>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Amount of payment</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Principal payment</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Interest payment</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Balance owed</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="result_layout">
     <item>
      <widget class="QLabel" name="total_info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Total paid:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="debt_info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Debt paid:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="interest_info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>Interest paid:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tax_layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
